### PR TITLE
V1.7.0rctestbranch

### PR DESCRIPTION
**Pull request by jliunyu**
1.7.0 RC with librdkafka v1.7.0-RC8
Since met the CI environment issue, the V1.7.0rc branch is not clean, hope this branch is Okay as a 1.7.0 RC.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
 # Confluent's Python client for Apache Kafka
 
+## v1.7.0
+
+v1.7.0 is a maintenance release with the following fixes and enhancements:
+
+- Add note about Windows environment variables (@slominskir, #1094).
```
Comment (jliunyu) [inline CHANGELOG.md:7]: I missed the review for last PR, will remove this when create the final release tag

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+- Add error_cb to confluent_cloud.py example (#1096).
+- Clarify that doc output varies based on method (@slominskir, #1098).
+- Docs say Schema when they mean SchemaReference (@slominskir, #1092).
+- Add document for NewTopic and NewPartitions (#1101).
+
+confluent-kafka-python is based on librdkafka v1.7.0, see the
+[librdkafka release notes](https://github.com/edenhill/librdkafka/releases/tag/v1.7.0)
+for a complete list of changes, enhancements, fixes and upgrade considerations.
```
Comment (jliunyu) [inline CHANGELOG.md:15]: ditto

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
 
+## v1.7.0
+
+v1.7.0 is a maintenance release with the following fixes and enhancements:
+
+- Add note about Windows environment variables (@slominskir, #1094).
+- Add error_cb to confluent_cloud.py example (#1096).
+- Clarify that doc output varies based on method (@slominskir, #1098).
+- Docs say Schema when they mean SchemaReference (@slominskir, #1092).
+- Add document for NewTopic and NewPartitions (#1101).
```
Comment (jliunyu) [inline CHANGELOG.md:11]: ditto

**Review by edenhill — APPROVED** [review]
LGTM, given that previous comments are fixed.

**Pull request closed without merging**